etcdctlv3/README.md

```diff
@@ -108,3 +108,41 @@ OK
 1
 ./etcdctl range foo
 ```
+
+
+### Watch [options] [key or prefix]
+
+Watch watches on keys or prefixes. Watch command never ends until it meets an error or user kills it.
```

Comment:
The watch command runs until it encounters an error or is terminated by the user.